pkg/util/idgen/idgen.go

```diff
@@ -14,6 +14,12 @@ const (
 
 	// NodeIDPrefix is the prefix of node ID.
 	NodeIDPrefix = "n-"
+
+	// TaskNamePrefix is the prefix of a system generated task name.
+	TaskNamePrefix = "t-name-"
+
+	// JobNamePrefix is the prefix of a system generated job name.
+	JobNamePrefix = "j-name-"
```

Comment:
Yeah, me either, I just needed something to generate a name if one wasn't provided. What is the expected behavior if the user doesn't provide a name? The `bacalhau job run` command will fail, should `docker run` and `wasm run` do the same? Note, this would be a change in their existing behavior.

Reply:
You are right about job name. This PR should help https://github.com/bacalhau-project/bacalhau/pull/4005
For task, use `main`